packages/footer/package.json

```diff
@@ -0,0 +1,76 @@
+{
+	"name": "@gov.au/footer",
+	"version": "0.1.0",
+	"description": "Footers help users who reach the bottom of a page without finding what they want.",
+	"keywords": [
+		"uikit",
+		"pancake-module",
+		"scss",
+		"css",
+		"gov.au",
+		"Commonwealth of Australia",
+		"design guide",
+		"design system",
+		"footer",
+		"footer-navigation",
+		"navigation"
+	],
+	"scripts": {
+		"postinstall": "pancake batter \"$(cd .. && npm prefix)\" -v",
```

Comment:
I don't think lerna let's you generate the package.json yet?
It's a fair point though. I would love to centralise this. For now it would be search & replace.